README.md

```diff
@@ -133,6 +133,16 @@ ChartMogul.Plan.all(config, query)
 ChartMogul.Plan.destroy(config, uuid)
 ```
 
+#### [Plan Groups](https://dev.chartmogul.com/docs/plan_groups)
```

Comment:
this actually doesn't exist still

Reply:
not a problem, you can merge on all client libraries once it's deployed ;) We'll just coordinate the release.